components/Navbar.js

```diff
@@ -0,0 +1,82 @@
+const Navbar = () => (
+  <div className="Navigation">
+    <div className="SoS-Text">
+      <h1>SUMMER OF SHIPPING</h1>
+      <img src="Ship Logo.svg" className="SoS-logo" />
+    </div>
+    <div className="Tabs">
+      <a href="/index">
+        HOME
+      </a>
+
+      <a href="https://forms.gle/3N3CfqbH6YkM1DF67">
+        WIKI
+      </a>
+
+      <a href="https://forms.gle/3N3CfqbH6YkM1DF67">
+        PROJECTS
+      </a>
+
+      <a href="https://forms.gle/3N3CfqbH6YkM1DF67">
+        MATCH MAKING
+      </a>
```

Comment:
Let's remove these two for now

Reply:
Finished this task as well, will be removed on next commit